languages/csharp/exercises/concept/structs/.docs/after.md

```diff
@@ -0,0 +1,37 @@
+C# `struct`s are closely related `class`s. They have state and behavior. They can have the same kinds of members, constructors, methods, fields, properties, etc fields and properties can be simple types, `struct`s and reference types. `struct`s observe the same rules about scope, read/write rules and access levels.
+
+One of the main things to remember is that when one struct is assigned to a variable or passed as a parameter the values are copied across so changes to he original variable will not affect the copied one and vice versa. In summary, `struct`s are **value types**.
+
+This [article][structs-patterns] discusses the differences between `struct`s and `class`s. You will see from the article that `struct`s tend to be lightweight and [immutable][structs-immutable] although this guidance is not enforced by the compiler or runtime.
```

Comment:
You can create `readonly` structs nowadays, but that is probably best left for the `readonly` exercise?

Reply:
Agreed.  that will give it a bit more meat.